.ci.yaml

```diff
@@ -2440,7 +2425,6 @@ targets:
 
   - name: Linux_pixel_7pro native_assets_android
     recipe: devicelab/devicelab_drone
-    presubmit: false
```

Comment:
Is this intended to enable in presubmit?

Reply:
That line was removed by mistake. Thanks!